demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -692,6 +697,15 @@ def validate_no_missing_release_notes(self, modified_files, old_format_files, ad
                                                                                    FileType.TEST_PLAYBOOK,
                                                                                    FileType.TEST_SCRIPT,
                                                                                    FileType.DOC_IMAGE})
+        if 'ApiModules' in packs_that_should_have_new_rn:
+            if not os.path.isfile(self.id_set_path):
+                IDSetCreator(print_logs=False, output=self.id_set_path).create_id_set()
```

Comment:
didn't you create it in 623? why again or am I missing something?

Reply:
technically yes, but actually no. 
line 623 is for `validate_changed_packs_unique_files`
line 702 is for `validate_no_missing_release_notes`

so no, but they are both called from `run_validation_using_git` - which calls 623 before 702, so technically yes